chatgpt_robusta_actions/chat_gpt.py

```diff
@@ -10,7 +10,7 @@
 lru_cache = cachetools.LRUCache(maxsize=cache_size)
 class ChatGPTTokenParams(ActionParams):
     """
-    :var token: ChatGPT auth token
+    :var chat_gpt_token: ChatGPT auth token
```

Comment:
This is not backward compatible. 
Any way we can support existing users?

Reply:
It doesn't actually change the parameter name. It just silences a warning written to the logs until now because the docstring documents a parameter (`token`) that doesn't exist.